src/software/ai/hl/stp/play/halt_play/BUILD

```diff
@@ -0,0 +1,64 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@simulated_tests_deps//:requirements.bzl", "requirement")
+
+cc_library(
+    name = "halt_play",
+    srcs = [
+        "halt_play.cpp",
+        "halt_play_fsm.cpp",
+    ],
+    hdrs = [
+        "halt_play.h",
+        "halt_play_fsm.h",
+    ],
+    deps = [
+        "//shared:constants",
+        "//software/ai/hl/stp/tactic/halt:halt_tactic",
+        "//software/logger",
+        "//software/util/generic_factory",
+    ],
+    alwayslink = True,
+)
+
+cc_test(
```

Comment:
you can delete the `halt_play_cpp_test`, we have enough coverage with `halt_play_test.py` and `halt_play_fsm_test.cpp`